src/query.js

```diff
@@ -94,7 +95,12 @@ function createFieldAliases (stats) {
 }
 
 function createStatFeatures (stats) {
+  stats = _.cloneDeep(stats)
+  console.log(stats)
   return stats.map(attributes => {
+    for (var key in attributes) {
+      if (key === 'dateField') attributes[key] = new Date(attributes[key]).getTime()
```

Comment:
this is too specific, we need to detect not fields called `dateField` but fields that are either of type: `Date` or are an ISO String